### DISCUSSION TITLE
Will be custom templates added like in version 2.x?

### DISCUSSION THREAD

**Discussion by ghost** [sni/thruk#1267] [Q&A] [answered]
Hi, congratulations for your good work.
As a daily user I would like to know if you are going to implement the custom templates per theme, like in the previous version.
I would like to rearrange the interface components/cards, without loosing all changes with the next update.

If not, can you add a basic id for each card, making custom css more easy to apply?

Thank you very much.

**Comment by sni** [comment, marked as answer]
There are no plans to do so, however, you can still override template from the `user_template_path` or from loaded plugins.
But i would ssi and javascript/css to do such things. And if you have good ideas about the layout, maybe it's useful for others as well. Which things would you like to rearrange?

**Comment by ghost** [comment]
Ok, thanks for your answer.
I'm not a dev, but I have users who enter to review their servers/services info, and I would like to move panels for a better visual experience (only desktop browser), moving the 2 panels on the left (_infobox.tt and the view links) to the right column under "command box", and make the service/host information (and performance chart)  2/3 witdth of the view.
I can do it by changing the tailwindcss class, and move div position in _tt files, but as said in a next update, i will loose that changes.
In the previous version (2.x) I have moved that parts of code to a minimal _tt and importing in the main.
Like in the screenshoot.
Thanks for your time.

![screenshot](https://user-images.githubusercontent.com/61549534/229108023-86ebce22-084a-451e-b676-568d1666f0e0.png)

**Comment by dan-m-joh** [comment]
I like the Idea to utilize the screen real estate more "efficient" and make "main-information" and "graph" use more space.

-- 
Dan

**Comment by sni** [comment]
i am all open for improvements :-)
